### ISSUE TITLE
Set django-settings-module in config file

### ISSUE THREAD

**Issue by hwalinga** [PyCQA/pylint-django#297]
I want to set the django-settings-module in the config file. I am not sure if I just do something wrong, or this just isn't supported yet. 

I tried to set it like this:

```py
[tool.pylint.master]
load-plugins = ["pylint_django"]
django-settings-module = "demap.settings"
```

**Comment by kontsaki** [comment]
it may be related to this problem i encountered
```
❯ pylint --load-plugins pylint_django --django-settings-module=settings_dev
Usage: pylint [options]

pylint: error: no such option: --django-settings-module`

```

**Comment by kontsaki** [comment]
@hwalinga try installing from git master, it works

**Comment by Zumarta** [comment]
How long does it take to get this feature into the pip-repository?

**Comment by atodorov** [comment]
Should be fixed with v2.4.0. 

> How long does it take to get this feature into the pip-repository?

As long as somebody figures out to request a new release. The original feature is related to command line, not immediately obvious that the 2 are related.
